sysdeps/managarm/generic/file.cpp

```diff
@@ -163,7 +163,7 @@ int sys_chroot(const char *path) {
 	return 0;
 }
 
-int sys_mkdir(const char *path) {
+int sys_mkdir(const char *path, mode_t) {
 	return sys_mkdirat(AT_FDCWD, path, 0);
```

Comment:
We should probably pass the mode parameter to mkdirat here, even though it's currently unused.